addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -349,7 +357,7 @@ def _save_incoming_attachment_fattura_pa(self, proxy_user, id_transaction, filen
 
         if self.env['ir.attachment'].search([('name', '=', filename), ('res_model', '=', 'account.move')], limit=1):
             # name should be unique, the invoice already exists
-            _logger.info('E-invoice already exists: %s', filename)
+            _logger.info("E-invoice '%s' is duplicated.", filename)
```

Comment:
"already exists" doesn't specify that it's an exception situation, even if it's not a warning.